File: Server/Stackless/src/Field.py
# Field.py
# Stackless Python Server

class Field:
    """
    Store data about a person.  Each Field has a name and has data.
    """
    
    def __init__(self, fieldName, fieldData):
        """
        Create a field with initial data.
        @param fieldName Name of the field.
        @param fieldData Data contained within the field.
        """
        self.name = str(fieldName);
        self.data = str(fieldData);
    
    def setName(self, newFieldName):
        """
        Set the new name for this field.
        @param newFieldName Name to set.
        """
        self.name = newFieldName;
    
    def getName(self):
        """
        Obtain the name of the field.
        @return String containing the name of the field.
        """
        return self.name;
    
    def setData(self, newFieldData):
        """
        Set new data for this field.
        @param newFieldData Data to set.
        """
        self.data = newFieldData;
    
    def getData(self):
        """
        Obtain the data contained within the field.
        @return String containing this field's data.
        """
        return self.data;

File: Server/Java/src/wowServ/wowBL/Field.java
package wowServ.wowBL;

/**
 * A Field contains any data about a person or thing.
 */
public class Field {
	/**
	 * Create a new field.
	 * 
	 * @param _name
	 *            Name of the field.
	 * @param data
	 *            Data contained in the field of any type.
	 */
	public Field(String _name, String _data) {
		setName(_name);
		setData(_data);
	}

	/**
	 * Set the name of this field.
	 */
	public void setName(String _name) {
		name = _name;
	}

	/**
	 * Get the name of this field.
	 */
	public String getName() {
		return name;
	}

	/**
	 * Set the data object stored in this field.
	 */
	public void setData(String _data) {
		data = _data;
	}

	/**
	 * Get the data stored in this field.
	 */
	public String getData() {
		return data;
	}

	private String name; // Name of the field.
	private String data; // Data that this field contains.
}
